src/twisted/mail/newsfragments/9733.bugfix

```diff
@@ -0,0 +1 @@
+twisted.mail._cred.CramMD5ClientAuthenticator.challengeResponse(...) now calls hmac.HMAC.new(...) with explicit MD5 digestmod.
```

Comment:
1. I would probably call this one a `misc` because it's about the internal implementation working on python 3.8, not something that's really interesting to users reviewing the release notes; the top-line item there is about a supported platform, not this specific API.
2. News fragments should always refer to public API names of things.  This one appears to be illustrating a bug in our process, because it *should* be showing up as public in `smtp` and `imap4` modules via `__all__` re-exports.  I filed this as a bug here: https://github.com/twisted/pydoctor/issues/184 to try to track it down.